polaris.shopify.com/src/components/SandboxHelpDialog/SandboxDialog.tsx

```diff
@@ -0,0 +1,81 @@
+import {useRef} from 'react';
+import {Dialog} from '@headlessui/react';
+import styles from './SandboxDialog.module.scss';
+import Longform from '../Longform';
+import Button from '../Button';
+
+function SandboxHelpDialog({
+  isOpen,
+  setIsOpen,
+}: {
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+}) {
+  const videoRef = useRef(null);
+  return (
+    <Dialog
+      initialFocus={videoRef}
+      open={isOpen}
+      onClose={() => setIsOpen(false)}
+    >
+      <div className={styles.PreventBackgroundInteractions}></div>
+      <div className="styles-for-site-but-not-polaris-examples">
+        <Dialog.Panel className={styles.DialogPanel}>
+          <p style={{textAlign: 'right'}}>
+            <Button
+              small
+              className={styles.CloseButton}
+              onClick={() => setIsOpen(false)}
+            >
+              ╳
+            </Button>
+          </p>
+          <div
+            className={styles.SandboxVideo}
+            // @ts-expect-error Why doesn't TS support CSS variables?
```

Comment:
Doing so would separate the definition of the source (`/images/sandbox-usage.mp4`), and its metadata (the aspect ratio), leading to increased effort when making changes (refactoring in particular could suffer as things get moved further apart or the video is replaced with a different aspect ratio, etc). This was an intentional choice to avoid separating on incorrect lines.